home_work_1/mains.py

```diff
@@ -0,0 +1,40 @@
+# print string "I love Python"
+
+count = 0
+
+while count < 42:
+    print("I love Python")
+    count += 1
+
+
+# creating a variable with the number of months
+
+age_in_month = 348
+
+# creating a variable with the number of years
+
+age_in_years = int(age_in_month / 12)
+
+# Creating a variable with text
+my_age = "Му name is Olexsandr. I’m " + str(age_in_years) + " years old"
+
+
+# Creating a variable with value = 1 and its comparison
+
+value = 1
+
+first_check = value > 5    # false
+second_check = value < 10  # true
+third_check = value == 12  # false
+four_check = value == 1    # true
+five_check = value <= 0    # false
+
+# Creating a variables
+
+a = "2"
```

Comment:
a, b and c initially should be integers, and d should be a string. Please, rewrite this part.